.github/workflows/formatting.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
   # formatting and basic install on cpu-only machine
   unit-tests:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
Need to work around `ModuleNotFoundError: No module named 'lib2to3'` since this was removed in python 3.13

Reply:
Need to update yapf to work around this.

Reply:
yapf changes are included in #6721

Reply:
PR merged.